### PR TITLE
From DB to Client #closed

### PR DESCRIPTION
**Pull request by BenHurCreations**
JS script to take the items list from the DB and display it on our website

(will be updated with more data)

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var mysql = require('mysql');
+
+var con = mysql.createConnection({
+  host: "localhost",
+  user: "root",
+  password: "Shaigal5%",
```
Comment (simzacks) [inline script.js]: storing passwords in the repo is stronglyy discouraged.
In general, it is a good idea to not store the connection details in the source code, and this should be read from a configuration file.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+var mysql = require('mysql');
```
Comment (simzacks) [inline script.js]: where is this javascript file going to be used? 
It looks like a serverside js, but your other code was going to be in python?

**Review by simzacks — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (BenHurCreations) [reply script.js]: ok sim, how do you suggest to do it?
I'm not familliar with the process

File: script.js (hunk 2 above)
Comment (BenHurCreations) [reply script.js]: it is going to be used in the website that Oded built, to display the data to the user

Comment (simzacks) [reply script.js]: Generally the client doesn't speak directly to the database.
You should be using an ORM, either with flask or django which talks to the database and presents the data to the user interface.

File: script.js (hunk 1 above)
Comment (simzacks) [reply script.js]: First of all, I suggest not connecting to the database directly from the client-side javascript.
As part of your ORM, you should have a place to configure the credentials.

Comment (BenHurCreations) [reply script.js]: Ok, I'd like to schedule with you and talk about how to do it best for the project
Let's coordinate in slack :)

**Comment by BenHurCreations** [comment]
Irrelevant for now so I close this request

**Pull request closed without merging**